src/include/rbd/librbd.h

```diff
@@ -1476,6 +1490,20 @@ CEPH_RBD_API int rbd_group_snap_list(rados_ioctx_t group_p,
 CEPH_RBD_API int rbd_group_snap_list_cleanup(rbd_group_snap_info_t *snaps,
                                              size_t group_snap_info_size,
                                              size_t num_entries);
+CEPH_RBD_API int rbd_group_snap_list2(rados_ioctx_t group_p,
+                                      const char *group_name,
+                                      rbd_group_snap_info_v2_t *snaps,
+                                      size_t *num_entries);
+CEPH_RBD_API int rbd_group_snap_list2_cleanup(rbd_group_snap_info_v2_t *snaps,
+                                              size_t num_entries);
+CEPH_RBD_API  int  rbd_group_snap_info(
```

Comment:
Let's include a verb in the name:
```suggestion
CEPH_RBD_API int rbd_group_snap_get_info(
```